chia/cmds/farm_funcs.py

```diff
@@ -113,13 +113,15 @@ async def get_wallets_stats(wallet_rpc_port: int) -> Optional[Dict[str, Any]]:
         wallet_client = await WalletRpcClient.create(self_hostname, uint16(wallet_rpc_port), DEFAULT_ROOT_PATH, config)
         amounts = await wallet_client.get_farmed_amount()
     except Exception as e:
-        if isinstance(e, aiohttp.ClientConnectorError):
-            print(f"Connection error. Check if wallet is running at {wallet_rpc_port}")
-        else:
-            print(f"Exception from 'wallet' {e}")
+        #
+        # Don't print anything here as errors are expected in many situations
+        # Just retrow the exception and let the caller handle it
+        #
+        raise e
```

Comment:
it seems redundant to catch the exception, just to re-throw it. You could just remove the whole `except` block